src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const yetanotherRobot = {};
```

Comment:
```suggestion
  const yetanotherRobot = {};
```
use camelCase for variables naming and meaningful one